src/components/elements/card.js

```diff
@@ -1,10 +1,59 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import styled from 'styled-components'
 
-const Card = ({ text }) => <article>{text}</article>
+const CardWrapper = styled.article`
+    width: 328px;
+    min-height: 356px;
+    box-sizing: border-box;
+    border-radius: 4px;
+    box-shadow: 0 16px 20px 0 rgba(0, 0, 0, 0.1);
+    background-color: var(--color-white);
+    padding: 4rem 5.6rem 4.6rem 5.6rem;
+    position: relative;
+    margin: 0 1rem;
+    ${props => props.firstCard && 'margin: 0 1rem 0 0'};
+    ${props => props.lastCard && 'margin: 0 0 0 1rem'};
+
+    div {
+        max-width: 216px;
+        bottom: 4.6rem;
+        margin-top: 4rem;
+
+        p {
+            font-size: 16px;
+            line-height: 24px;
+            margin-top: 8px;
+        }
+        span {
```

Comment:
Think you can use `Header` component `as="h2"` here

Reply:
as `h4` is better I think.